packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -265,11 +269,45 @@ const PRESETS = [
             ['project.commerce.shortCode']: 'kv7kzm78',
             ['project.commerce.isSlasPrivate']: false,
             ['project.einstein.clientId']: '1ea06c6e-c936-4324-bcf0-fada93f83bb1',
-            ['project.einstein.siteId']: 'aaij-MobileFirst'
+            ['project.einstein.siteId']: 'aaij-MobileFirst',
+            ['project.demo.enableDemoSettings']: false
         },
         assets: ['translations'],
         private: false
     },
+    {
+        id: 'retail-react-app-demo-private',
+        name: 'Retail React App Demo Private Client',
```

Comment:
🤔 the word "private" here refers to the private client. But I think more importantly, it's also meant to be used privately by us. 

How about tweak the name to something like `retail-react-app-demo-internal`?

Reply:
The preset name has been changed to `retail-react-app-demo-site-internal`